server/routes/courtRegister/courtRegisterController.ts

```diff
@@ -93,7 +93,7 @@ export default class CourtRegisterController {
     const view = new AddNewCourtDetailsView(
       req.session.addNewCourtForm,
       courtTypes,
-      req.session.courtListPageLink,
+      req.session.courtListPageLink as string,
```

Comment:
needs further investigation on why this is nullable, since it isn't defined as that